scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityCreationTemplate.java

```diff
@@ -250,10 +260,15 @@ private Optional<SchedulingActivityDirective> createInstanceForReal(final String
         throw new UnsupportedOperationException("Activity creation duration arguments cannot depend on simulation results.", e);
       }
       duration.ifPresent(d -> tnw.addDurationInterval(name, d, d));
+      storedDur = duration.get();
     }
     final var success = tnw.solveConstraints();
     if (!success) {
-      logger.warn("Inconsistent temporal constraints, will try next opportunity for activity placement if it exists");
+      logger.warn("Inconsistent static temporal constraints, will try next opportunity for activity placement if it exists");
+      logger.info("Start range " + this.startRange);
+      logger.info("Start range " + this.endRange);
+      if(storedDur!=null) logger.info("dur range " + storedDur);
+      logger.info("interval range " + interval);
```

Comment:
The "End range" on L269 is a typo. The change to `DEBUG` level is an ignorable suggestion being offered because the information doesn't seem to have value for a non-debugging user (at least based on the way it is presented).
```suggestion
      logger.debug("Start range " + this.startRange);
      logger.debug("End range " + this.endRange);
      if(storedDur!=null) logger.debug("dur range " + storedDur);
      logger.debug("interval range " + interval);
```